Variables: Add support for `$__timezone` template variable (#66785)
grafana/grafana: docs/sources/dashboards/variables/add-template-variables/index.md

- SQL queries in MySQL, Postgres, and MSSQL.
- The `$__timeFilter` variable is used in the MySQL data source.

### $\_\_timezone

> The `$__timezone` variable returns the currently selected time zone, either `utc` or an entry of the IANA time zone database (e.g. `America/New_York`).

Reviewer: We avoid Latin abbreviations.
```suggestion
The `$__timezone` variable returns the currently selected time zone, either `utc` or an entry of the IANA time zone database (for example, `America/New_York`).
```

> The `$__timezone` variable returns the currently selected time zone, either `utc` or an entry of the IANA time zone database (e.g. `America/New_York`).

Reviewer: Is there any specific reason that _UTC_ isn't capitalized on line 320?

> If the currently selected time zone is _Browser Time_, your browser time zone will be guessed.

Reviewer: ```suggestion
If the currently selected time zone is _Browser Time_, Grafana will try to determine your browser time zone.
```
